<launch>

  <node pkg="kingfisher_experiments" type="velocity_tracker.py" name="velocity_tracker" output="screen">

    <!-- <remap from="~cmd_drive" to="/cmd_drive"/> -->
    <remap from="~odom" to="/pose_gt"/> <!-- /imu/odometry | /pose_gt -->

    <param name="send_goal_button" value="7"/>
  </node>

</launch>
